package/gluon-config-mode-geo-location-osm/Makefile

```diff
@@ -13,7 +13,11 @@ define Package/gluon-config-mode-geo-location-osm/install
 	$(Gluon/Build/Install)
 
 	$(INSTALL_DIR) $(1)/lib/gluon/config-mode/www/static/
-	$(LN) /lib/gluon/web/www/static/gluon-web-osm.js $(1)/lib/gluon/config-mode/www/static/
```

Comment:
This is incorrect and won't build without `CONFIG_GLUON_MINIFY` - the file is in `gluon-web-osm`, `gluon-config-mode-geo-location-osm` only creates a link. I don't think `gluon-config-mode-geo-location-osm` needs to be changed in this PR. Same thing for `gluon-config-mode-core` - just have `gluon-web-model` install one or the other file, without changing the creation of the config-mode link.

Reply:
First part I missed, thanks.
Second was by choice, so that we'd keep the `.min.js` in `/lib/gluon/web/www/static/`.
But I'm indifferent, will change that there as well.

Reply:
I don't think we should leave the `.min.js` for any files in `/lib/gluon/web/www/static/`, so other packages' symlinks don't need to depend on `CONFIG_GLUON_MINIFY`.

Reply:
this is hopefully resolved and outdated since ed3551a5ca734556f861f4c62ffb199f2830ad44